tools/mksurfdata_map/src/mkpftMod.F90

```diff
@@ -509,24 +513,21 @@ subroutine mkpft(ldomain, mapfname, fpft, ndiag, allow_no_crops, &
 
      call domain_checksame( tdomain, ldomain, tgridmap )
 
-     ! Area-average percent cover on input grid [pctpft_i] to output grid 
-     ! [pctpft_o] and correct [pctpft_o] according to land landmask
-     ! Note that percent cover is in terms of total grid area.
-  
-     do no = 1,ns_o
-        pctlnd_o(no)     = tgridmap%frac_dst(no) * 100._r8
-        ldomain%frac(no) = tgridmap%frac_dst(no) 
-     end do
+     mask_src(:) = tdomain%mask(:)
```

Comment:
As in mklaiMod, I wonder if it's really necessary to have this extra temporary variable, mask_src. I don't have strong feelings about it, though, so am fine with keeping it if you prefer it.

Reply:
[See this response.](https://github.com/ESCOMP/CTSM/pull/823#discussion_r339716302)